src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTransactionTest {
```

Comment:
We do not need to test POJO classes like FruitTransaction. They can be excluded from jacoco.